docs/validator.md

```diff
@@ -99,13 +99,13 @@ docker compose up --detach
 
 ## Obtaining your Hugging Face token
 
-The dataset for code, `The Stack V1-dedup`, requires a **Hugging Face access token**. Follow these steps to obtain and configure one:
+The dataset for code, `The Stack V2-dedup`, requires a **Hugging Face access token**. Follow these steps to obtain and configure one:
```

Comment:
Can you also emphasize that it also needs AWS secret and access keys that can be obtained for free, so they need to read carefully through both step 1 and step 2.